openid-sharedsignals-eventdef-specification-1_0.md

```diff
@@ -0,0 +1,282 @@
+---
+title: OpenID Shared Signals Event Definition Specification
+abbrev: SSFEvents-Spec
+docname: openid-sharedsignals-eventdef-specification-1_0
+date: 2024-06-06
+
+ipr: none
+cat: std
+wg: Shared Signals
+
+coding: us-ascii
+pi:
+  toc: yes
+  sortrefs: yes
+  symrefs: yes
+  private: yes
+
+author:
+      -
+        ins: T. Cappalli
+        name: Tim Cappalli
+        org: Okta
+        email: tim.cappalli@okta.com
+      -
+        ins: J. Schreiber
+        name: Jen Schreiber
+        org: Workday
+        email: jennifer.winer@workday.com
+      -
+        ins: A. Deshpande
+        name: Apoorva Deshpande
+        org: Okta
+        email: apoorva.deshpande@okta.com
+normative:
+  OpenID.Core:
+    author:
+    - ins: N. Sakimura
+      name: Nat Sakimura
+    - ins: J. Bradley
+      name: John Bradley
+    - ins: M.B. Jones
+      name: Michael B. Jones
+    - ins: B. de Medeiros
+      name: Breno de Medeiros
+    - ins: C. Mortimore
+      name: Chuck Mortimore
+    date: November 2014
+    target: http://openid.net/specs/openid-connect-core-1_0.html#IDToken
+    title: OpenID Connect Core 1.0 - ID Token
+  RFC2119:
+  RFC8174:
+  RFC5280:
+  RFC5646:
+  RFC6711:
+  RFC8176:
+  SSF:
+    target: http://openid.net/specs/openid-sse-framework-1_0.html
```

Comment:
This is referencing a very out of date draft. Please update to draft 3: https://openid.net/specs/openid-sharedsignals-framework-1_0-03.html